sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseCreateForm.java

```diff
@@ -203,7 +205,11 @@ protected void addFields() {
 		facilityTypeGroup.setWidth(100, Unit.PERCENTAGE);
 		facilityTypeGroup.addItems(FacilityTypeGroup.getAccomodationGroups());
 		getContent().addComponent(facilityTypeGroup, FACILITY_TYPE_GROUP_LOC);
-		facilityType = addField(CaseDataDto.FACILITY_TYPE);
+		facilityType = new ComboBox();
```

Comment:
Why was this change needed? Is it related to the bug that you were writing about? Because technically this should do exactly what the line that you removed is doing

Reply:
Precisely. That's what I thought too and that's also why I spend several hours searching the bug. This line definitely needs to be changed.